.github/ISSUE_TEMPLATE/patch_release.md

```diff
@@ -18,31 +18,32 @@ Please assign the release manager to the issue.
 
 This issue can be closed when we have completed the following steps (in order).
 Please ensure all artifacts (PRs, workflow runs, Tweets, etc) are linked from
-this issue for posterity. Refer to this [prior release issue][release-1.7] for
-examples of each step.
+this issue for posterity. Refer to this [prior release issue][release-1.11.1] for
+examples of each step, assuming vX.Y.Z is being cut.
 
 - [ ] Updated all version information in the documentation on the relevant release branch.
```

Comment:
From https://github.com/crossplane/crossplane/pull/3702#discussion_r1117488438, I think this can be removed too - the docs have nothing at all that needs to be done for a patch release 😅